script.js

```diff
@@ -0,0 +1,72 @@
+
+//Function to check the validity of a password
+const isValidFunction = (passWord, userName)=>{
```

Comment:
```suggestion
const isValidPassword = (passWord, userName)=>{
```
@GoutamShanbhag here `isValidPassword` is a better name for this function as the name immediately tell us what the function is performing.